_posts/2020/08/2020-08-31-governance.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Governance"
+date: 2020-08-31
+---
+
+I had another good conversation with [Mike Hoye](http://exple.tive.org/blarg/) this week
+about life, the Internet, and everything.
+I always learn a lot talking with him,
+and what I realized this time is that *governance should be discoverable*.
+Pretty much every open source project has a `LICENSE.md` file in their root directory these days,
+and most now have a `CONDUCT.md` file as well.
+These files tell passers-by how they're allowed to use the software
+and what social norms they're required to adhere to if they want to contribute to it.
+
+But how can they find out who decides what the project is going to do next,
+or who gets a say in deciding that?
+Large organizations like the [Python Software Foundation](https://www.python.org/psf/)
+[typically have bylaws](https://arxiv.org/abs/2005.10063)---as registered non-profits,
+they're legally required to---but
+[most smaller don't explain any of this](https://opensource.com/open-organization/18/4/new-governance-model-research).
+They may have a `CONTRIBUTING.md` file,
+but that usually describes how to set up a build environment and how to file bug reports,
+not how decisions are made or by whom.
+
+Spelling this out facilitates
+[legitimate peripheral participation](https://en.wikipedia.org/wiki/Legitimate_peripheral_participation),
+which [helps projects grow](https://journals.plos.org/ploscompbiol/article?id=10.1371%2Fjournal.pcbi.1007296).
+More importantly,
+it's essential to making a project genuinely inclusive.
+As Jo Freeman pointed out fifty years ago in her influential essay
+"[The Tyranny of Structurelessness](https://www.jofreeman.com/joreen/tyranny.htm)",
+every organization has a power structure:
+the only question is whether it's accountable or not.
+If a group says it doesn't have rules,
+what it's really saying is that it's run by an old boys' network,
+and that getting things done depends on your personal connections with unofficial influencers
+and on how self-confident and assertive you are.
+If you didn't room with one of the core developers in college
+and aren't willing to barge into discussions,
+you are often effectively voiceless.
+
+I would therefore like every open project to have a file in its root directory called `GOVERNANCE.md`
+that explains:
+
+1.  Who gets a say in which decisions
+    (where "gets a say" means "must approve" or "can veto"---promises of consultation
+    are a social nicety without such guarantees).
+2.  How to get a decision made.
+3.  How to find out what decisions have been made.
+4.  How to become part of the decision-making process (e.g., how to get a vote).
+5.  How to change these rules (i.e., how to amend the constitution).
+
```

Comment:
Something that seems to be missing is the "when" (i.e., when a decision will be made? How long I have to wait until my PR is accepted/rejected?). Or maybe it is implicitly included in point 2?

Reply:
1. "When" depends on how much time people have and how many other issues are in play. I personally dislike adding "as soon as possible under the circumstances" because it doesn't actually tell people anything...

2. No disrespect to Mikael, but I think people who have power generally like the idea of loose rules more than people without power. If a project wants to officially adopt a policy of "everything passes unless someone specifically objects", they can, but they still need to spell out quite clearly how to tell that an issue is being debated, how to register an objection, and how to tell if you're one of the people whose objections count, all of which gets us back to "strict".

Reply:
I really believe that it is all about transparency (it does not matter if the rules are loose or not). If the rules are there, and the way the project works is clear and accessible (including what to do if you want to propose a change), people can easily understand and decide if they want to move ahead or not. Any a priori information when you are joining a project is a pro. 
@gvwilson, I missed a point related to what is the vision of the project (long-term and short-term definitions). Or is it part of 3?